src/bulma.tsx

```diff
@@ -259,15 +245,12 @@ export function removeLoadingProps(props: Bulma.Loading) {
     return rest;
 }
 
-export function getColorModifiers({ isColor: color }: Bulma.Color) {
-    return isColor(color) ? { [`is-${color}`]: true } : {};
+export function getColorModifiers(color: Bulma.Color) {
```

Comment:
you need to change `Bulma.color` to `string`:
```
export function getColorModifiers(color: Bulma.Color) {
...
```

since this function is going to return something like `is-white` or `is-my-custom-color`.